project/src/main/world/environment/stool.gd

```diff
@@ -1,10 +1,14 @@
+tool
 extends "res://src/main/world/environment/overworld-obstacle.gd"
 """
 A stool which appears on the overworld.
 
 The stool can toggle to an 'occupied' state when it's sat upon. This affects its collisions and appearance.
 """
 
+export (Texture) var occupied_texture: Texture
+export (Texture) var unoccupied_texture: Texture
```

Comment:
Add comments.

Reply:
Added comments; `the texture to use when the stool has a creature sitting on it`